libmc/_client.pyx

```diff
@@ -384,14 +385,15 @@ cdef class PyClient:
         self._imp = new Client()
         self._imp.config(CFG_HASH_FUNCTION, hash_fn)
         rv = self._imp.init(c_hosts, c_ports, n, c_aliases)
+        PyMem_Free(c_hosts)
+        PyMem_Free(c_ports)
+        PyMem_Free(c_aliases)
+
         if failover:
             self._imp.enableConsistentFailover()
         else:
             self._imp.disableConsistentFailover()
 
-        PyMem_Free(c_hosts)
```

Comment:
These 3 lines are moved to earlier lines. Why?

Reply:
@mckelvin see https://github.com/douban/libmc/commit/5968657268cfca8fc472491b6b1475a9451a22a9 also

Reply:
because @everpcpc's pr https://github.com/douban/libmc/pull/74/files#diff-721f906017b85cb2e3838d0775a3138dR418 already do this, i'll revert this change。